Automattic/wp-calypso
# A4A: Add the 'Growth Accelerator' card on the Overview sidebar.

This pull request involves the implementation of the Growth Accelerator card, which will be added to the Overview sidebar page. This new feature aims to provide users with a tool to boost their growth and productivity within the application.

<img width="713" alt="Screenshot 2024-10-08 at 7 13 30 PM" src="https://github.com/user-attachments/assets/60e8ada3-61e1-49ef-93ce-f5a2b073cd53">


Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/1082

## Proposed Changes
* Implement the OverviewSidebarGrowthAccelerator card and add it to the Overview sidebar.
* Make the new card dismissible based on the `a4a_growth_accelerator_dismissed` preference.
## Why are these changes being made?

* This allows the users to schedule a 'Growth accelerator' session with our support team speeding up the process.
## Testing Instructions

* Use the A4A live link and go to the `/overview` page.
* Confirm that you see the 'Growth Accelerator' card.
* Confirm that the CTA button redirects to the correct link.
* Confirm that Clicking the 'Not interested' button dismisses the card.

## Pre-merge Checklist
